lib/rules/no-jquery.js

```diff
@@ -27,62 +31,85 @@ module.exports = {
   ERROR_MESSAGE,
 
   create(context) {
-    let destructuredAssignment;
-    let emberImportAliasName;
-    let jqueryImportAliasName;
-    const report = function (node) {
+    let report = function (node) {
       context.report(node, ERROR_MESSAGE);
+      // Only report this error message once
+      report = () => {};
```

Comment:
The main difference between this approach and the original, was now the error message gets reported multiple times. Up to 3 times??

1. for an `ImportDeclaration` with value of `'jquery'` eg `import $ from 'jquery'`
2. for a variable in scope for a `$` or `jQuery` definition
3. I'm unsure about..